vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/SequenceOverviewJsonFormat.java

```diff
@@ -0,0 +1,44 @@
+package edu.asu.diging.vspace.core.services.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import edu.asu.diging.vspace.core.model.SequenceOverview;
+import edu.asu.diging.vspace.core.model.impl.Sequence;
+
+@Component
+public class SequenceOverviewJsonFormat {
```

Comment:
this class seems like it should be somewhere in the web package. Its only job is to create an object that can be turned into json, right? It's only used in the controllers?

Reply:
I have removed this class and merged the methods into SequenceOverviewManager